src/RealtimeClient.ts

```diff
@@ -40,6 +40,7 @@ export type RealtimeClientOptions = {
   fetch?: Fetch
   worker?: boolean
   workerUrl?: string
+  accessTokenCallback?: () => Promise<string>
```

Comment:
I'd call it `accessToken`, just like the param in auth config for SupabaseClient

Reply:
will change 👍